recipe/build.sh

```diff
@@ -19,7 +19,7 @@ eval export ${LIBRARY_SEARCH_VAR}="${PREFIX}/lib"
 # Build LAPACK.
 # Enable threading. This can be controlled to a certain number by
 # setting OPENBLAS_NUM_THREADS before loading the library.
-make QUIET_MAKE=1 DYNAMIC_ARCH=1 BINARY=${ARCH} NO_LAPACK=0 NO_AFFINITY=1 USE_THREAD=1 CFLAGS="${CF}"
+make QUIET_MAKE=1 DYNAMIC_ARCH=1 BINARY=${ARCH} NO_LAPACK=0 NO_AFFINITY=1 USE_THREAD=1 CFLAGS="${CF}" FFLAGS="-frecursive"
```

Comment:
Need `-frecursive` to fix the problem as explained in this [comment]( https://github.com/xianyi/OpenBLAS/issues/539#issuecomment-92135151 ). However, `FCOMMON_OPT` results in `gfortran` flags being overridden/dropped that shouldn't be. Switching to `FFLAGS` seems to fix the problem.